src/Type/Definition/ListOfType.php

```diff
@@ -4,24 +4,38 @@
 
 namespace GraphQL\Type\Definition;
 
+use GraphQL\Type\Schema;
+use function is_callable;
+
 class ListOfType extends Type implements WrappingType, OutputType, NullableType, InputType
 {
     /** @var Type */
```

Comment:
this can also be callable according to modified constructor

Reply:
True enough! a8f870d